### PR TITLE
Add more BigQuery topics.

### PR DESCRIPTION
**Pull request by deflaux**

### CODE REVIEW

File: docs/source/use_cases/annotate_variants/annovar.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+__ RenderedVersion_
+
+.. comment: end: goto-read-the-docs
+
+
+If your source data is single-sample VCF, `gVCF`_, or Complete Genomics masterVar format, this page offers some solutions to annotate all variants found within the cohort using `Annovar`_ or similar tools.
+
+(1) First, load your data into Google Genomics and export your variants to BigQuery.  See `Load Genomic Variants`_ for more detail as to how to do this.
+
+(2) Note that merging has occured during the import process, so each unique variant within the cohort will be a separate record within the variant set, with all calls for that variant nested within the record.  For more information see `Variant Import merge logic details`_.
```
Comment (pgrosu) [inline docs/source/use_cases/annotate_variants/annovar.rst]: s/occured/occurred

File: docs/source/use_cases/analyze_variants/gwas.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+`Google BigQuery`_ can be used to perform a GWAS.  Here are several examples:
+
+* Chi-squared tests on :doc:`/use_cases/discover_public_data/1000_genomes` dataset with members of EAS super population as case and control all other populations:
+
+  * iPythonNotebook `Genome-wide association study (GWAS).ipynb <https://github.com/googlegenomics/datalab-examples/blob/master/datalab/genomics/Genome-wide%20association%20study%20(GWAS).ipynb>`_
+  * SQL `gwas-pattern-chi-squared-test.sql <https://github.com/googlegenomics/bigquery-examples/blob/master/1000genomes/sql/gwas-pattern-chi-squared-test.sql>`_
+
+* Two-proportion Z test on :doc:`/use_cases/discover_public_data/1000_genomes` dataset with members of EAS super population as case and control all other populations:
+
+  * SQL `gwas-pattern-two-proportion-z-test.sql <https://github.com/googlegenomics/bigquery-examples/blob/master/1000genomes/sql/gwas-pattern-two-proportion-z-test.sql>`_
```
Comment (pgrosu) [inline docs/source/use_cases/analyze_variants/gwas.rst]: In the SQL it says:

```
# z-score critical value for p-value=5*10^-8 is +/-5.326724`
# > qnorm(1 - 5e-08)
# [1] 5.326724
```
But this is a two-sided test, so the p-value would need to be divided in half:

```
> qnorm(1 - 5e-08/2)
[1] 5.45131
```
So the following would need to be updated in the query:
```
z_score >= 5.326724
  OR z_score <= -5.326724
```

File: docs/source/use_cases/annotate_variants/interval_joins.rst [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+    Alt,
+    Func,
+    Gene,
+    PopFreqMax,
+    ExonicFunc,
+  ORDER BY
+    Chr,
+    Start
+
+The Tute table is for SNP annotation.  If a different table were used to annotate both SNPs and INDELs, you may wish apply a similar UDF to also "bin" the rows from the variants table, since INDELs may span multiple bins.
```
Comment (mbookman) [inline docs/source/use_cases/annotate_variants/interval_joins.rst]: s/wish apply/wish to apply/

File: docs/source/use_cases/load_data/multi_sample_variants.rst [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+.. code::
+
+    WHERE
+      reference_name = 'chr13'
+      AND start <= 33628137
+      AND end >= 33628138
+
+to capture not only that variant, but any other records that overlap that genomic position.
+
+Suppose we want to calculate an aggregate for a particular variant, such as the number of samples with the variant on one or both alleles and of samples that match the reference?  The WHERE clause above will do the trick.  But then suppose we want to do this for all SNPs in our dataset?
```
Comment (mbookman) [inline docs/source/use_cases/load_data/multi_sample_variants.rst]: Nit: this paragraph double-spaces after periods. Other paragraphs use the more modern single-space after paragraph.

File: docs/source/use_cases/load_data/multi_sample_variants.rst [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+* Use a cluster computing job to transform data with non-variant segments to variant-only data with calls from non-variant-segments merged into the variants with which they overlap. This is currently done only for SNP variants. Indels and structural variants are left as-is.
+
+  * see the `Dataflow example <https://github.com/googlegenomics/codelabs/tree/master/Java/PlatinumGenomes-variant-transformation>`_
+  * see the `Hadoop Streaming example <https://github.com/googlegenomics/codelabs/tree/master/Python/PlatinumGenomes-variant-transformation>`_
+
+* Use BigQuery to transform the data, materialize the result to a new table
+
+  * https://github.com/googlegenomics/bigquery-examples/tree/master/pgp/data-stories/schema-comparisons#motivation
+
+A note about scaling: as the number of samples increases, so does the number of private and rare variants.  At a certain point there are many, many rows with mostly 0/0 genotypes.  We are experimenting with alternate transformations.  Comment on `this issue <https://github.com/googlegenomics/codelabs/issues/52>`_ if you want a pointer to the most recent prototype.
```
Comment (mbookman) [inline docs/source/use_cases/load_data/multi_sample_variants.rst]: Nit: this paragraph double-spaces after periods. Other paragraphs use the more modern single-space after paragraph.

**Comment by mbookman** [comment]
LGTM

**Pull request merged**